### ISSUE TITLE
Property 'label' does not exist on type 'string'.

### ISSUE THREAD

**Issue by helixquar** [helixquar/asciidecorator#6]
The error:

```
Property 'label' does not exist on type 'string | T'.
  Property 'label' does not exist on type 'string'.
```
In file: extension.ts
On the ff. block:

```typescript
    let options = <vscode.QuickPickOptions>{
        placeHolder: 'Select a font',
        matchOnDescription: true,
        onDidSelectItem: item => {
            if(!txt)
            {
                return;
            }

            fontTestWindow.clear();
            fontTestWindow.append(
                figlet.textSync(txt, {
                font: item.label,
                horizontalLayout: 'default',
                verticalLayout: 'default'
            }));
            fontTestWindow.show(true);
        }
    };
```

**Comment by helixquar** [comment]
Explicitly declaring the type with:

```diff
+        onDidSelectItem: <QuickPickItem>(item:QuickPickItem) => {
-        onDidSelectItem: item => {
```

Produces:

Property 'label' does not exist on type 'QuickPickItem'.

**Comment by helixquar** [comment]
Fixed as of b6e548e227fbe274de7b04982cfcfe01730cbf5d by adding interface.
